pan3d/explorers/xr-globe.py

```diff
@@ -0,0 +1,526 @@
+import vtk
```

Comment:
file name should just be `globe.py` and you should import each class from vtkmodules.xxx instead of full vtk.